tests/InstantTest.php

```diff
@@ -448,60 +448,60 @@ public function providerCompareTo() : array
             [-1,  1,  1, -1, -1],
             [-1,  1,  1,  0, -1],
             [-1,  1,  1,  1, -1],
-            [ 0, -1, -1, -1,  1],
```

Comment:
We'll have fun discussing this rule in brick/math :)

https://github.com/brick/math/blob/ec750c1b5bf4158b3823309bbe1887b40e837c8d/tests/BigIntegerTest.php#L2329-L2389

(spoiler: once settled on a coding standard, I'd like to apply it to all projects)

Not sure how we can reconcile readability & coding standards in large tests like these. Thoughts?